include/avif/avif.h

```diff
@@ -523,20 +523,20 @@ typedef struct avifCodecConfigurationBox
 struct avifIO;
 
 // Destroy must completely destroy all child structures *and* free the avifIO object itself.
-// This function pointer is optional, however, if the reader isn't intended to be owned by
```

Comment:
At this point, it is not clear what "reader" refers to. Also, avifIO is also designed for writing (not implemented yet). So I changed "reader" to "avifIO object".